cloud-common/src/main/resources/application.yml

```diff
@@ -53,7 +53,7 @@ vault:
 
 cb:
   uaa.startup.timeout.sec: 300
-  enabledplatforms: AZURE,AWS,GCP,OPENSTACK
+  enabledplatforms: AZURE,AWS,GCP,OPENSTACK,MOCK
```

Comment:
We don't want to allow `MOCK` as i know

Reply:
Yes. Fixed, and added a unit test for that.